app/views/content/funding-and-support.md

```diff
@@ -2,9 +2,9 @@
 title: "Teacher training funding"
 heading: "Fund your teacher training"
 description: |-
-    Explore funding for teacher training, including student finance, bursaries and scholarships, and support if you’re disabled, a parent or carer or a veteran.
+    Explore funding for teacher training, including tuition fees, student finance, bursaries and scholarships, and support if you’re disabled, a parent or carer or a veteran.
 title_paragraph: |-
-    If you're applying for teacher training in England, we can help you understand what funding and practical support you could be eligible for.
+    If you're applying for teacher training in England, we can help you understand how much it may cost and what funding and practical support you could be eligible for.
```

Comment:
I didn't think it was misleading as I used may to try and caveat that as it may not cost them anything. And we do something similar with the funding and say could be eligible for as salaried courses aren't eligible for student finance. But I can reword if you think it needs to be clearer.

Reply:
Ignore this - this is the category page so what you have written makes sense (think I was reading it as the fees page!)